ibis/sql/compiler.py

```diff
@@ -1011,6 +1017,11 @@ def _get_keyword_list(self):
         return ["INTERSECT" for _ in range(len(self.tables) - 1)]
 
 
+class Difference(SetOp):
+    def _get_keyword_list(self):
+        return ["EXCEPT" for _ in range(len(self.tables) - 1)]
```

Comment:
This should be the same, but clearer, right?
```suggestion
        return ["EXCEPT"] * (len(self.tables) - 1)
```

Reply:
Yes I could change it to that